pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -743,6 +743,18 @@
     private String kinitCommand = "/usr/bin/kinit";
 
     /**** --- BookKeeper Client --- ****/
+    @FieldContext(
+        category = CATEGORY_STORAGE_BK,
+        doc = "BookKeeper ledgers storage connection string when using a separated BookKeeper cluster."
+            + " If not set it will use local zookeeper quorum of Pulsar cluster"
+    )
+    private String bookkeeperLedgersStore;
```

Comment:
in bookkeeper we deprecated using two separate settings. we are encouraging people to use metadata service uri to connect to a zookeeper cluster. we should try to adopt it in Pulsar. Thus if we are adding the support for a separate bookkeeper cluster, let's try to use metadata service uri.

Reply:
done, thanks.